lib/watir/logger.rb

```diff
@@ -22,26 +22,36 @@ class Logger
 
     def_delegators :@logger, :debug, :debug?,
                    :info, :info?,
-                   :warn, :warn?,
+                   :warn?,
                    :error, :error?,
                    :fatal, :fatal?,
                    :level
 
-    def initialize
+    def initialize(progname="Watir")
       @logger = create_logger($stdout)
+      @logger.progname = progname
+      @ignored = []
+    end
+
+    def ignore(id)
```

Comment:
I would suggest supporting multiple arguments. Right now, to ignore multiples, you would have to do:

~~~~~~~~
Watir.logger.ignore(:class_array)
Watir.logger.ignore(:always_locate)
~~~~~~~~

It would be nice to be able to do:

~~~~~~~~
Watir.logger.ignore(:class_array, :always_locate)
~~~~~~~~

Reply:
I implemented as an array instead of args